src/Makefile

```diff
@@ -5,6 +5,8 @@ KDIR := /lib/modules/$(KERNELVERSION)/build
 PWD := $(shell pwd)
 EXTRA_CFLAGS := -g
 
+all:	default
```

Comment:
https://stackoverflow.com/questions/2514903/what-does-all-stand-for-in-a-makefile

It's tradition, and some build systems (ie the one I use) run "make all" when you tell it to build and it doesn't work if there's no all in there.